src/Module.php

```diff
@@ -24,6 +24,8 @@ public function getConfig(): array
 
     /**
      * Register a specification for the HydratorManager with the ServiceListener.
+     *
+     * @deprecated ModuleManager support will be removed in version 5.0 of this component
```

Comment:
```suggestion
     * @deprecated Since 4.17.0 This method is not necessary for module manager and will be removed in 3.0
```

You can still use the module manager but the interface from above and this method are not necessary for this.

Reference:

- https://docs.laminas.dev/laminas-modulemanager/module-manager/#servicelistener
- https://github.com/laminas/laminas-modulemanager/blob/3cd6e84ba767b43a47c6c4245a56b30ac3738c6a/src/Feature/HydratorProviderInterface.php#L9-L18
- https://docs.laminas.dev/laminas-filter/v3/migration/v2-to-v3/#removal-of-the-filterproviderinterface

Reply:
Thanks for pointing that out! 🙏 I’ve just changed it from v3 to v5